src/sagas/index.tsx

```diff
@@ -1,11 +1,11 @@
-import { takeLatest } from 'redux-saga/effects';
-import { REQUEST_FETCH, REQUEST_DELETE } from '../actions';
-import { fetchData, deleteData } from './Post';
+// import { takeLatest } from 'redux-saga/effects';
+// import { REQUEST_FETCH, REQUEST_DELETE } from '../actions';
+// import { fetchData, deleteData } from './Post';
 
-export function* rootSaga() {
-  yield [
-    takeLatest(REQUEST_FETCH, fetchData),
-    takeLatest(REQUEST_DELETE, deleteData),
-    // takeLatest(REQUEST_CREATE, createData),
-  ];
-}
+// export function* rootSaga() {
+//   yield [
+//     takeLatest(REQUEST_FETCH, fetchData),
+//     takeLatest(REQUEST_DELETE, deleteData),
+//     // takeLatest(REQUEST_CREATE, createData),
+//   ];
+// }
```

Comment:
Please delete code instead of  comment out, because comment out causes unreadable code and you can restore code easily thanks to Git.

You should comment out only comment, not code.